packages/frontend-v2/components/Sidebar/Sidebar.tsx

```diff
@@ -80,6 +90,13 @@ const Sidebar: React.FC<SidebarProps> = memo(({ major }) => {
       >
         {major.name}
       </Text>
+      <Box padding="10px 20px 15px 20px">
```

Comment:
Might also be worth putting this in a collapsible portion of the sidebar. I think from design perspective this might make more sense but might want to ask Evelyn what she thinks.